### ISSUE TITLE
[Question] - Is it possible to customize the consumer-group-id of an agent ?

### ISSUE THREAD

**Issue by abusi** [robinhood/faust#630]
Hi all,

I have a use case were I want to consume some topic using a given consumer group id and some other for which I want another consumer-group-id. Sadly, I'm not able to do so in Faust, as the consumer-group-id for agent is derived from the `faust.app(id=str)` parameter.

Would it be possible to make agent customize their consumer_id using a ctor parameter like:

```
@app.Agent("an_agent", consumer_group_id="a_consumer_id")
async def my_agent(stream):
    pass

@app.Agent("another_agent",  consumer_group_id="another_consumer_group_id")
async def another_agent(stream):
    pass
``` 

OR if you have any pointer on how to achieve this kind of use cases, i'm all hears

Regards,

**Comment by g-clef** [comment]
Can you make multiple "apps" and have each one with a different `id` that corresponds to the consumer group you want them in?

eg:
```
app1 = faust.App("consumer_group1")
app2 =faust.App("consumer_group2")

@app1.Agent("a_topic")
async def agent1(stream):
   # do stuff
   pass

@app2.Agent("a_topic")
async def agent2(stream):
  # do other stuff
  pass
```

**Comment by abusi** [comment]
Yes, i can do that, but how can I start them both ? `app1.add_dependency(app2)` then `app1.main()` ? will it work ?
IMHO, it is not really elegant :/

**Comment by g-clef** [comment]
You can make a "worker" and have the worker class start all the apps together. Have a look at https://readthedocs.org/projects/faust/downloads/pdf/latest/ and search for "Multiple apps" (Sorry, can't deep-link into the PDF).
